README.md

```diff
@@ -43,14 +42,18 @@ $ cat << EOF > config.yaml
   - staging
   - prod
 EOF
+```
 
+```text
```

Comment:
this change is to not use the `bash` highlighting for text output (otherwise certain words in the text output such as `in` get highlighted).